### PR TITLE
updates tiddit

### PR DESCRIPTION
**Pull request by jemten**
### This PR adds | fixes:

- Tiddit version 3.4.0 improves the positioning of the SV calls as well as introducing a basic scoring algorithm. The improved positioning means that we try to switch back to sv a sv annotation overlap of 0.8

#. ## How to test:

- Automatic and continuous test suite

### Expected outcome:
- [ ] Installation, unit and integration test suite pass

### Review:
- [ ] Code review
- [ ] Tests pass

This [version](https://semver.org/) is a:
- [ ] **MAJOR** - when you make incompatible API changes. If applicable record manual test results in PR header
- [ ] **MINOR** - when you add functionality in a backwards compatible manner. If applicable record manual test results in PR header
- [ ] **PATCH** - when you make backwards compatible bug fixes or documentation/instructions

### CODE REVIEW

File: lib/MIP/Constants.pm [hunk 1]
```diff
@@ -82,7 +82,7 @@ Readonly our %ANALYSIS => (
 );
 
 ## Set MIP version
-Readonly our $MIP_VERSION => q{11.1.1};
+Readonly our $MIP_VERSION => q{11.1.1-develop};
```
Comment (henrikstranneheim) [inline lib/MIP/Constants.pm:85]: Hmm, that is  a new one

**Review by henrikstranneheim — COMMENTED** [review]
Looks good so far

File: lib/MIP/Constants.pm (hunk 1 above)
Comment (jemten) [reply lib/MIP/Constants.pm:85]: yes, I we need a way to indicate that this is a develop version when you install on stage. We could call it something else of course :)

Comment (jemten) [reply lib/MIP/Constants.pm:85]: Turned out that it needed to be numeric for the command `mip version` to work. I'll change it

**Comment by jemten** [comment]
Tested on two validation samples 
| Case | MIP 11.0 position  | MIP 11.1 position | New position |
| ----- | ----------------- | ----------------- | ------------- |
| Brightcaiman | 21:27030970 - 21:28056597 |  21:27030970 - 21:28056427 | 21:27031095 - 21:28056526 |
| Onemite |   6:129655051 - 6:129670154 | 6:129655051 - 6:129670083 | 6:129655051 - 6:129670083 |
| intentcorgi  | 11:103016482 - 11:103177264 | 11:103016481 - 11:103177264 | 11:103016481 - 11:103177264 |
| __safeguinea__ | __20:57437612__ | __20:57437521__ | __20:57437614__ |

Roughly same position for for 3 out of 4. The fourth sample have improved positioning with the new version

**Review by henrikstranneheim — APPROVED** [review]
👍

**Pull request merged**